README.md

```diff
@@ -268,4 +284,5 @@ When using rails, you can generate a model with all the fields you have on your
  [1]: http://www.salesforce.com
  [2]: https://github.com/ejholmes/restforce
  [3]: https://github.com/bkeepers/dotenv
+ [4]: https://developer.salesforce.com/docs/atlas.en-us.api_asynch.meta/api_asynch/bulk_api_2_0.htm
```

Comment:
readme is helpful. I see you've listed the link to the SF Bulk API doc, would it be helpful to explicitly call our the limits of this API here?

Reply:
I'd rather link to Salesforce as the source of truth than to keep our README up to date with their documentation.